src/schema/notifications.ts

```diff
@@ -302,8 +322,12 @@ export class ZimbraNotifications {
 		if (items) {
 			items.forEach((i: any) => {
 				const item = normalizeFolder(i);
+				let itemId = item.id;
```

Comment:
const itemId = i.id.includes(':') ? this.findSharedItemId(i.id) : item.id; ?

Reply:
Done.